enginetest/enginetests.go

```diff
@@ -2871,15 +2871,11 @@ func TestWarnings(t *testing.T, harness Harness) {
 			Query: `
 			SHOW WARNINGS LIMIT 0
 			`,
-			Expected: []sql.Row{
-				{"", 3, ""},
-				{"", 2, ""},
-				{"", 1, ""},
-			},
+			Expected: nil,
```

Comment:
These were just wrong. AFAICT, MySQL treats `LIMIT 0` on a SHOW WARNINGS by not returning any of the warnings.

Reply:
That would make sense!